src/runtime-rs/crates/agent/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub mod kata;
 mod log_forwarder;
 mod sock;
 pub mod types;
+use protocols::agent::DropCacheRequest;
```

Comment:
The agent related Request should be better defined in 'types', which is under 'crates/agent/src/type.rs' now, like the other Request.
This situation maybe changed in the future, as we do not have to support grpc right now. But I think It might be easier to do some same modifications by then.